src/dcrypto/v1/v1_test.go

```diff
@@ -92,3 +94,37 @@ func TestRoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func TestHash(t *testing.T) {
+	sizes := []int{24, 1024, 15872, 16364, 16384, 16394, 16896, 66560}
+	for _, size := range sizes {
+		h := md5.New()
```

Comment:
I thought drive's hash of the remote files were md5 so just used that but I can use something else just so people don't see it and assume the worst.